core/node/state_keeper/src/updates/l1_batch_updates.rs

```diff
@@ -14,6 +14,7 @@ pub struct L1BatchUpdates {
     pub txs_encoding_size: usize,
     pub l1_tx_count: usize,
     pub finished: Option<FinishedL1Batch>,
+    pub rolling_txs_hash: Option<H256>,
```

Comment:
why `Option`? even if there are no txs, the default is H256::zero, so it could be used

Reply:
It may make sense for a batch to have "None" hash when it was not calculated becouse we did not have precommits applied. I would leave this.

Reply:
agree, for updates it makes sense